hledger-lib/Hledger/Utils/String.hs

```diff
@@ -338,17 +336,7 @@ takeWidth w (c:cs) | cw <= w   = c:takeWidth (w-cw) cs
 -- (not counted), and line breaks (in a multi-line string, the longest
 -- line determines the width).
 strWidth :: String -> Int
-strWidth = maximum . (0:) . map (foldr (\a b -> charWidth a + b) 0) . lines . stripAnsi
-
--- | Strip ANSI escape sequences from a string.
---
--- >>> stripAnsi "\ESC[31m-1\ESC[m"
--- "-1"
-stripAnsi :: String -> String
-stripAnsi s = either err id $ regexReplace ansire "" s
- where
-   err    = error "stripAnsi: invalid replacement pattern"      -- PARTIAL, shouldn't happen
-   ansire = toRegex' "\ESC\\[([0-9]+;)*([0-9]+)?[ABCDHJKfmsu]"  -- PARTIAL, should succeed
+strWidth = maximum . (0:) . map (foldr (\a b -> charWidth a + b) 0) . lines
```

Comment:
Utils is sort of a grab bag of helpers that may come in handy - I don't mind having a few things in there that aren't currently being used. Maybe it's worth keeping stripAnsi and a strWidthAnsi in there for now ?

Reply:
Fine by me. Should I amend the commit to add it back in, or re-add it in a new commit on the end?